conf/common/requirements.txt

```diff
@@ -11,3 +11,6 @@ pytest
 pandas
 yapf
 git+https://github.com/SymbiFlow/edalize.git@symbiflow#egg=edalize
+git+https://github.com/facebookresearch/hydra.git#egg=hydra-core
```

Comment:
There are dev releases you can use if you want to use Hydra 1.1 (e.g. https://pypi.org/project/hydra-core/1.1.0.dev5/).
Depending directly on master sounds like a bad idea.

Let me know if you are depending on something in master than is not yet in the latest dev release.